ast/rego.peg

```diff
@@ -156,9 +156,9 @@ DefaultRules <- "default" ws name:Var _ "=" _ value:Term {
 
     loc := currentLocation(c)
     body := NewBody(NewExpr(BooleanTerm(true).SetLocation(loc)))
-    body[0].Location =loc
```

Comment:
The expression should also have a location set. I'd leave this line and just add the one below.